helm/csi-isilon/values.yaml

```diff
@@ -374,7 +374,7 @@ authorization:
 # Enable this feature only after contact support for additional information
 podmon:
   enabled: false
-  image: dellemc/podmon:v1.3.0
+  image: dellemc/podmon:v1.4.0
```

Comment:
@alikdell hope you have executed resiliency tests once with sidecar update.

Reply:
> @alikdell hope you have executed resiliency tests once with sidecar update.

@nitesh3108 Please elaborate your concerns or you want us to test any specific way. version 1.4.0 will be available only after we release that version.

Reply:
Hi @nitesh3108, yes, our integration tests with PowerScale have been executing against the latest changes for Resiliency.

Reply:
Noted